soccer/gameplay/tactics/wall.py

```diff
@@ -25,7 +25,8 @@ def __init__(
                 0, 0),  # what point we are defending (default is goal)
             defender_spacing=2.5,  # number of robot radii between the centers of the defenders in the wall
             dist_from_mark=.75,  # distance from the mark point we want to build the wall
-            defender_priorities=[20, 19, 18, 17, 16]
+            defender_priorities=[20, 19, 18, 17, 16],
+            contest_ball=False
     ):  # default defense priorities                       
         super().__init__(continuous=True)
 
```

Comment:
for is_ball_free, you may want to add a comment about what this actually means since it is a massive block of a lambda expression.

Reply:
I wish i knew what it means too

Reply:
PLEASE DEAR GOD REDO THE NEW LINES IN IS_BALL_FREE